src/hist/core.py

```diff
@@ -11,19 +11,26 @@
 class BaseHist(Histogram):
     def __init__(self, *args, **kwargs):
         """
-            Initialize Hist object. Axis params must contain the names.
+            Initialize BaseHist object. Axis params must contain the names.
```

Comment:
```suggestion
            Initialize BaseHist object. Axis params can contain the names.
```

Only `NamedHist` *requires* names. They are optional in BaseHist.

Reply:
A typo 😊.